testing/test.js

```diff
@@ -31,6 +31,15 @@ function getVendingItem(amount, item) {
       }
       return "Not enough money for Coca-Cola";
   }
+  switch (item) {
```

Comment:
This secondary switch case isn't necessary just add another case inside the existing switch.